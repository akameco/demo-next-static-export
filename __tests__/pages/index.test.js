// @flow
import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Comp from '../../pages/index'

test('snapshot', () => {
  const x = shallow(<Comp />)
  expect(toJson(x)).toMatchSnapshot()
})
