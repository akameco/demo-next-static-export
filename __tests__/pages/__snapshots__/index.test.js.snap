// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<main>
  <SideEffect(Head)>
    <title>
      Home page
    </title>
  </SideEffect(Head)>
  Hello World
</main>
`;
